pbf/audience/update.proto

```diff
@@ -3,5 +3,67 @@ syntax = "proto3";
 package audience;
 option go_package = ".;audience";
 
-message UpdateI {}
-message UpdateO {}
+// UpdateI is the input for updating audiences.
+//
+//     {
+//         "obj": {
+//             "metadata": {
+//                 "organization.venturemark.co/id": "<id>",
+//                 "user.venturemark.co/id": "<id>"
+//             },
+//             "jsnpatch": [
+//                 {
+//                     "ope": "add",
+//                     "pat": "/user",
+//                     "val": "xh3b4sd"
+//                 },
+//                 {
+//                     "ope": "remove",
+//                     "pat": "/user",
+//                     "val": "marcoelli"
+//                 }
+//             ]
```

Comment:
@marcusellison here an example of adding and removing a user within the audience resource's `user` property. Multiple operations in one request.